pkg/download/downloader.go

```diff
@@ -74,15 +74,16 @@ func extractZIP(targetDir string, read io.ReaderAt, size int64) error {
 		if err != nil {
 			return errors.Wrap(err, "can't create file in zip destination dir")
 		}
+		close := func() {
```

Comment:
I don't quite see why `defer` can't be used here.

Reply:
it would defer until all of the files are extracted.